elements/pfe-navigation/src/pfe-navigation.scss

```diff
@@ -64,6 +70,9 @@ $variables: (
   }
```

Comment:
on line 57 of this file, i think the breakpoint needs to be "md" instead of "sm"
so that the padding on the right & left edges is correct and matches up with content at that smaller (less than 768) size 

![image](https://user-images.githubusercontent.com/456863/72368831-cd12a180-36cc-11ea-88ba-4d2e41947848.png)